admin-tools/lib/src/main/scala/com/gu/mediaservice/ImageDataMerger.scala

```diff
@@ -16,13 +16,17 @@ import scala.concurrent.{Await, ExecutionContext, Future}
 object ImageDataMergerConfig {
   private val gridClient = GridClient(maxIdleConnections = 5)
 
-  def apply(apiKey: String, domainRoot: String): ImageDataMergerConfig = {
+  def apply(apiKey: String, domainRoot: String, imageLoaderEndpointOpt: Option[String]): ImageDataMergerConfig = {
```

Comment:
why Option?

Reply:
if not passed we use the one from Services class look at: https://github.com/guardian/grid/blob/d83b3e370b5c3e5fab53b62130467ec2bc1ecfa4/admin-tools/dev/app/controllers/AdminToolsCtr.scala#L16

i can change the var name `imageLoaderEndpointOpt` -> `alternativeImageLoaderEndpointOpt`

maybe it will make more sense then

Reply:
i can make it string and treat as restricted to Pass if you prefer ?